packages/pds-ember/addon/components/pds/button/stories/layouts/index.stories.js

```diff
@@ -109,25 +115,26 @@ const TextBetweenIcons = () => ({
     </Pds::Button>
   `,
   context: getContext(),
-});
-
-IconOnly_ariaLabel.story = { name: 'Icon Only using [aria-label]' };
-IconOnly_visuallyHidden_label.story = { name: 'Icon Only with visually-hidden label' };
-TwoIcons_withoutBlock.story = { name: 'Two Icons without block' };
-TwoIcons_withBlock.story = { name: 'Two Icon with block' };
-TextBeforeIcon.story = { name: 'Label + Icon' };
-TextAfterIcon.story = { name: 'Icon + Label' };
-TextBetweenIcons.story = { name: 'Icon + Label + Icon' };
+})
+TextBetweenIcons.storyName = 'Icon + Label + Icon'
 
-export {
-  CONFIG as default,
-
-  TextOnly,
-  IconOnly_ariaLabel,
-  IconOnly_visuallyHidden_label,
-  TwoIcons_withoutBlock,
-  TwoIcons_withBlock,
-  TextBeforeIcon,
-  TextAfterIcon,
-  TextBetweenIcons,
-}
+export const LengthyContent = () => ({
+  template: hbs`
+    <Pds::Button
+      @compact={{isCompact}}
+      @iconEnd="test-icon"
+      @iconStart="test-icon"
+    >
+      Text with lengthy content.
+      The quick brown fox jumps over the lazy dog.
+      The quick brown fox jumps over the lazy dog.
+      The quick brown fox jumps over the lazy dog.
+      The quick brown fox jumps over the lazy dog.
+      The quick brown fox jumps over the lazy dog.
+      The quick brown fox jumps over the lazy dog.
+      The quick brown fox jumps over the lazy dog.
+      The quick brown fox jumps over the lazy dog.
+    </Pds::Button>
+  `,
+  context: getContext(),
+})
```

Comment:
I had to add this test case in Storybook, so I cleaned up the file to use recommended syntax for Storybook 6 story definitions.